src/app/shell/admin-tools/data/import-export/export-providers/export-providers.component.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { of } from 'rxjs';
+import { AdminImportExportService } from 'shared/services/admin-import-export/admin-import-export.service';
+import { TranslateModule } from '@ngx-translate/core';
+import { ExportProvidersComponent } from './export-providers.component';
+
+describe('ExportProvidersComponent', () => {
+  let component: ExportProvidersComponent;
+  let fixture: ComponentFixture<ExportProvidersComponent>;
+  let adminImportExportService: jest.Mocked<AdminImportExportService>;
+
+  beforeEach(async () => {
+    const spy = {
```

Comment:
It is a very generic name.
If you are using it for AdminImportExportService, name it appropriately, e.g. adminImportExportServiceMock